plugins/integrations/cloudian/src/main/java/org/apache/cloudstack/cloudian/CloudianConnector.java

```diff
@@ -24,7 +24,7 @@
 public interface CloudianConnector extends PluggableService {
 
     ConfigKey<Boolean> CloudianConnectorEnabled = new ConfigKey<>("Advanced", Boolean.class, "cloudian.connector.enabled", "false",
-            "If set to true, this enables the Cloudian Connector for CloudStack.", true);
+            "If set to true, this enables the Cloudian Connector for CloudStack.", false);
```

Comment:
@davidjumani this won't work - maybe for UI but for the plugin to instantiate - you'll need to restart mgmt server, as it's used in the `configure` method of manager/class

Reply:
Exactly why it's being set as non dynamic. The user will get a notification that the MS needs to be restarted after this change and on restart it'll pick up the necessary API

Reply:
Ah I see, my bad I misunderstood - lgtm.